src/infrastructure/adapters/output/repositories/accounts/sqlalchemy/account_repository_sqlalchemy.py

```diff
@@ -0,0 +1,84 @@
+from typing import override
+
+from injector import singleton, inject
+from sqlalchemy import Engine, select
+from sqlalchemy.orm import Session
+
+from src.app.ports.output.repositories.account_repository import (
+    AccountRepository,
+)
+from src.domain.model.account import Account, AccountId
+from src.infrastructure.adapters.output.repositories.accounts.sqlalchemy.account_mapper import (
+    AccountMapper,
+)
+from src.infrastructure.adapters.output.repositories.accounts.sqlalchemy.account_model import (
+    Base,
+    AccountDao,
+    CellphoneDao,
+)
+
+
+@singleton
+class AccountRepositorySQLAlchemy(AccountRepository):
+    __engine: Engine
+
+    @inject
+    def __init__(self, engine: Engine) -> None:
+        self.__engine = engine
+        Base.metadata.create_all(engine)
```

Comment:
I don't know if this is the proper location of this metadata creation, or if it has to be defined in another location